main.cc

```diff
@@ -1007,7 +1007,9 @@ To start the scylla server proper, simply invoke as: scylla server (or just scyl
                 };
             });
             cm.start(std::move(get_cm_cfg), std::ref(stop_signal.as_sharded_abort_source())).get();
-            auto stop_cm = deferred_stop(cm);
+            auto stop_cm = defer_verbose_shutdown("compaction manager", [&cm] {
+                cm.stop().get();
```

Comment:
I think it's better to have compaction_manager::stop swallow any bad exception and not leak them to main.